src/applications/static-pages/authenticated-homepage/selectors.js

```diff
@@ -0,0 +1,5 @@
+import { toggleValues } from 'platform/site-wide/feature-toggles/selectors';
+import FEATURE_FLAG_NAMES from 'platform/utilities/feature-toggles/featureFlagNames';
+
+export const selectShowAuthenticatedHomepage = state =>
```

Comment:
Totally bike shedding here...  I know we've prefixed these feature flag selectors with `select` in the past, I wonder if calling it `showAuthenticatedHomepageFeatureFlag` makes more sense and would be clearer in use?